Ships as Raw.java.txt

```diff
@@ -0,0 +1,881 @@
+
+Name|Gladiator I-class Star Destroyer
+Type|Gladiator

```

Comment:
Might want to check the file name on this guy.

This looks like a duplicate file, and has a .java. tossed in the middle